modules/heat_conduction/src/bcs/GapHeatTransfer.C

```diff
@@ -204,6 +254,13 @@ GapHeatTransfer::computeQpJacobian()
          _phi[_j][_qp];
 }
 
+Real
+GapHeatTransfer::computeSlaveQpJacobian()
+{
+  return _test[_i][_qp] * -(*_slave_side_phi)[_slave_j][0] * _edge_multiplier *
```

Comment:
Shouldn't this have a term like the one on line 252?

Reply:
You would have to tell me that d_gap_conductance_d_Tmaster and d_gap_conductance_d_Tslave are equivalent before I felt comfortable adding that term

Reply:
They are equivalent.  The conductance is based on the average of the two temperatures, so a change on the master or slave side would have the same effect.  I suspect that including this might make the convergence even better on the LWR example that I mentioned.

Reply:
Ok, added